xtask/src/main.rs

```diff
@@ -206,6 +207,98 @@ fn run_host_tests(test_opt: &TestOpt) -> Result<()> {
     run_cmd(cargo.command()?)
 }
 
+/// Formats the project: nix, rust, and yml.
+fn run_fmt_project(fmt_opt: &FmtOpt) -> Result<()> {
+    // fmt rust
+    {
+        eprintln!("Formatting: rust");
+        let mut command = Command::new("cargo");
+        command.arg("fmt");
+        if fmt_opt.check {
+            command.arg("--check");
+        }
+        command
+            .arg("--all")
+            .arg("--")
+            .arg("--config")
+            .arg("imports_granularity=Module");
+
+        match run_cmd(command) {
+            Ok(_) => {
+                eprintln!("✅ rust files format")
+            }
+            Err(e) => {
+                if fmt_opt.check {
+                    eprintln!("❌ rust files to not pass check");
+                } else {
+                    eprintln!("❌ rust formatter failed: {e:#?}");
+                }
+            }
+        }
+    }
+
+    // fmt yml
+    if has_cmd("yamlfmt") {
+        eprintln!("Formatting: yml");
+        let mut command = Command::new("yamlfmt");
+        if fmt_opt.check {
+            command.arg("-lint");
+        }
+        command.arg(".");
```

Comment:
```suggestion
        command.arg(".github");
```

`yamlfmt` doesn't respect `.gitignore`, so when I ran it locally it picked up on yml files that another tool happened to have put under `target/`. (The tool is also noticeably slower on my machine when scanning the whole tree due to accumulated files under `target/`.)